src/config.rs

```diff
@@ -0,0 +1,37 @@
+use failure::{bail, Error, Fail, ResultExt};
+use std::env::VarError;
+use std::str::FromStr;
+use std::sync::Arc;
+
+#[derive(Debug)]
+pub struct Config {
+    pub(crate) max_file_size: usize,
+    pub(crate) max_file_size_html: usize,
+}
+
+impl Config {
+    pub fn from_env() -> Result<Self, Error> {
```

Comment:
The idea is that over time we'll move more of the environment variables here, instead of being spread out over the codebase? I like it, it makes things much easier to test as well.

Reply:
Yep, that's the goal!

Reply:
I completely support that, having a concrete storage of env vars in one place would be fantastic